guides/periodic_jobs.md

```diff
@@ -0,0 +1,119 @@
+  * `*/15` — Step, matches any value that is a multiple (0, 15, 30, 45)
+  * `0-5` — Range, matches any value within the range (0, 1, 2, 3, 4, 5)
+  * `0-9/2` — Step values can be used in conjunction with ranges (0, 2, 4, 6, 8)
+
+Each part may have multiple rules, where rules are separated by a comma. The allowed values for
+each field are as follows:
+
+| Field      | Allowed Values                               |
+| ---------- | -------------------------------------------- |
+| `minute`   | 0-59                                         |
+| `hour`     | 0-23                                         |
+| `days`     | 1-31                                         |
+| `month`    | 1-12 (or aliases, `JAN`, `FEB`, `MAR`, etc.) |
+| `weekdays` | 0-6 (or aliases, `SUN`, `MON`, `TUE`, etc.)  |
+
+### Cron Extensions
+
+Oban supports these common cron extensions:
+
+| Expression                 | Translates To                                   |
+| -------------------------- | ----------------------------------------------- |
+| `@hourly`                  | `0 * * * *`                                     |
+| `@daily` (or `@midnight`)  | `0 0 * * *`                                     |
+| `@weekly`                  | `0 0 * * 0`                                     |
+| `@monthly`                 | `0 0 1 * *`                                     |
+| `@yearly` (or `@annually`) | `0 0 1 1 *`                                     |
+| `@reboot`                  | Run once at boot time across the entire cluster |
+
+### Examples
+
+Here are some specific examples that demonstrate the full range of expressions:
+
+  * `0 * * * *` — The first minute of every hour
+  * `*/15 9-17 * * *` — Every fifteen minutes during standard business hours
+  * `0 0 * DEC *` — Once a day at midnight during December
+  * `0 7-9,4-6 13 * FRI` — Once an hour during both rush hours on Friday the 13th
+
+For more in depth information see the man documentation for `cron` and `crontab` in your system.
+Alternatively you can experiment with various expressions online at [Crontab Guru][guru].
+
+## Caveats & Guidelines
+
+  * All schedules are evaluated as UTC unless a different timezone is provided. See
+  `Oban.Plugins.Cron` for information about configuring a timezone.
+
+  * Workers can be used for regular _and_ scheduled jobs so long as they accept different arguments.
+
+  * Long-running jobs may execute simultaneously if the scheduling interval is shorter than it takes
+  to execute the job. You can prevent overlap by passing custom `unique` opts in the crontab config:
+
+  ```elixir
+  custom_args = %{scheduled: true}
+
+  unique_opts = [
+    period: 60 * 60 * 24,
+    states: [:available, :scheduled, :executing]
+  ]
+
+  config :my_app, Oban,
+    repo: MyApp.Repo,
+    plugins: [
+      {Oban.Plugins.Cron,
+       crontab: [
+         {"* * * * *", MyApp.SlowWorker, args: custom_args, unique: unique_opts}
+       ]}
+    ]
+  ```
```

Comment:
This is no longer true. As of Oban v2.18 cron doesn't use uniqueness at all, and for many versions it has avoided that problem by only scheduling at the top of the minute. Furthermore, only the leader schedules jobs.